boards/DCM/Src/App/states/App_FaultState.c

```diff
@@ -28,6 +28,17 @@ static void FaultStateRunOnEntry(struct StateMachine *const state_machine)
 
 static void FaultStateRunOnTick1Hz(struct StateMachine *const state_machine)
 {
+    struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+
+    // Clear inverter fault if requested by a PCAN node
+    App_SharedStates_HandleClearInvFaultsCmd(
+        App_DcmWorld_GetCanTx(world), can_rx);
+
+    // Open or close the inverter LV switches if requested by a PCAN node
+    App_SharedStates_ConfigInverterSwitches(
+        can_rx, App_DcmWorld_GetInverterSwitches(world));
+
     App_SharedStatesRunOnTick1Hz(state_machine);
```

Comment:
how do we transition out of fault state to the init state?

Reply:
I forgot to mention it recently but after setting one of the EEPROM parameters within the inverter (the skip precharge parameter), I noticed that we no longer fault directly on the application of LV if we don't have HV. Therefore, we never transition into the fault state unless an actual fault occurs.

Reply:
Let's leave what we have in the init state for turning on LV to the inverters since it doesn't hurt.